ztp/Makefile

```diff
@@ -1,17 +1,13 @@
-POLICYGEN_DIR := ./ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator
+.PHONY: ci-job build-policygen checkSourceCrs build-policygen-kustomize
 
-export GOFLAGS := $(if $(GOFLAGS),$(GOFLAGS),-mod=vendor)
+ci-job: build-policygen checkSourceCrs build-policygen-kustomize
```

Comment:
I really like this differentiation between `build-policygen` and `build-policygen-kustomize`  Very clean!

Reply:
Maybe from this level the targets should actually be called `test-policygen` and `test-policygen-kustomize`?

Reply:
okay